catalog.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>DLK</title>
+</head>
+<body>
+    <header>
+        <img src="images/Union.png" alt="">
+
+        <nav>
+            <ul class="menu">
+                <li>
+                    <a href="#" class="menu-link">BLOG</a>
+                </li>
+                <li>
+                    <a href="#" class="menu-link">CATALOG</a>
+                </li>
+                <li>
+                    <a href="#" class="menu-link">ABOUT US</a>
+                </li>
+            </ul>
+        </nav>
+        <a href="#" class="link-search">
+            <img src="images/magnifier.png" alt="">
+        </a>
+        <a href="#" class="link-contact-us">CONTACT US</a>
+    </header>
+
+<main>
+    <h1>Category Products</h1>
+    <h3>Home
+        <img src="images/Vector (Stroke).svg" alt="">
+        Catalog
+    </h3>
+    <section class="block-catalog-links">
+        <div>
+            <article>
+                <img src="images/kids.png" alt="">
+                <h3>
+                    <a href="#" class="APPAREL KIDS">APPAREL KIDS</a>
+                </h3>   
+            </article>
+            <article>
+                <img src="images/holiday.png" alt="">
```

Comment:
https://stackoverflow.com/questions/1140755/what-is-the-difference-between-src-images-logo-gif-and-src-images-logo-gif